vllm/engine/arg_utils.py

```diff
@@ -533,6 +534,9 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
         parser.add_argument('--enable-lora',
                             action='store_true',
                             help='If True, enable handling of LoRA adapters.')
+        parser.add_argument('--enable-lora-bias',
+                            action='store_true',
+                            help='If True, enable bias for LoRA adapters.')
```

Comment:
This might be a stupid question but wouldn't we want to always use the bias when the tensors exist for a particular adapter? Or is it the case that we need to know up front (since adapters might be loaded dynamically later)?

It's not ideal to have to configure this explicitly.

Reply:
Oh I just saw @Yard1's original comment for this ... yeah I guess there isn't a straightforward perfect solution, so maybe how you have it is the best option.